core/examples/luigi-example-next/README.md

```diff
@@ -0,0 +1,31 @@
+# Sample Luigi application written in NextJS
+
+## Overview
+
+This is the NextJS-based sample application which runs with Luigi framework.
```

Comment:
```suggestion
This is the NextJS-based sample application which runs with Luigi framework.
Due to the nature of NextJS's routing strategy, it is not possible to use `useHashRouting: false` in the Luigi Configuration. If you want to use path based routing, Luigi Core's index.html should be served separately.
```

Maybe something like this? So people know about the fact and could do something about it by themself, if they want.